lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -158,6 +158,26 @@ function dosomething_northstar_get_northstar_user($drupal_id) {
 }
 
 /**
+ * Checks if a Northstar user exists for a given index & type.
+ * @param  string $index_type The type of index to query for (Email, Facebook, Mobile, _id)
+ * @param  string $index The index to query for
+ * @return bool|object
```

Comment:
Yeah I ended up changing it to a bit of both, I can rename it & return object|null